tests/test.js

```diff
@@ -13,17 +13,26 @@ var c = new GoogleContacts({
 c.getContacts(function (err, contacts) {
   if (err) throw err;
   assert.ok(typeof contacts === 'object', 'Contacts is not an object');
-  console.log(contacts);
+  console.log('Showing first 10 contacts out of '+ contacts.length)
+  console.log(contacts.slice(0,9));
```

Comment:
sorry, forgot to remove this... I wanted a cleaner version of the results

before merging I'll return this to original log